src/main/java/uk/gov/pay/api/service/LedgerService.java

```diff
@@ -118,6 +120,27 @@ public SearchRefundsResponseFromLedger searchRefunds(Account account, Map<String
         throw new SearchRefundsException(response);
     }
 
+    public SearchDisputeResponseFromLedger searchDispute(Account account, Map<String, String> paramsAsMap) {
+        paramsAsMap.put(PARAM_ACCOUNT_ID, account.getAccountId());
+        paramsAsMap.put(PARAM_TRANSACTION_TYPE, DISPUTE_TRANSACTION_TYPE);
+
+        Response response = client
+                .target(ledgerUriGenerator.transactionsURIWithParams(paramsAsMap))
+                .request()
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .get();
+
+        if (response.getStatus() == SC_OK) {
+            try {
+                return response.readEntity(SearchDisputeResponseFromLedger.class);
+            } catch (ProcessingException exception) {
+                throw new SearchRefundsException(exception);
```

Comment:
Should this be SearchDisputesException ?

Reply:
👍